README.md

```diff
@@ -20,6 +20,22 @@ This is useful if you want to leave timing markers in place for future use, but
 
 The feature `"const_array"` will use a const array rather than a vec for the thread local storage of traces.
 
-The feature `"lfence"` will add an lfence instruction before and after each call to rdtsc.
+The feature `"lfence"` will add an lfence instruction before and after each call to rdtsc (x86 only).
 
 Run e.g. `cargo bench --features "tsc-trace/capacity_1_million"` to show the runtime overhead difference between using this library, vs directly calling rdtsc twice and subtracting.
+
+## Viewer
+
+Dependency on SDL2 (https://crates.io/crates/sdl2) and bytemuck (https://crates.io/crates/bytemuck).
+
+A visual representation of cycles gathered by tsc-trace.
+
+Takes a file that has traces written to it through command line arguments, format:
```

Comment:
Maybe worth specifying the file needs to be in the format used by write_traces_binary

Reply:
Thanks! Updated README to mention this.